spdk/src/poller.rs

```diff
@@ -0,0 +1,156 @@
+use std::{
```

Comment:
We already have implementation of SPDK poller in Rust:
```
src/core/poller.rs
```

Reply:
I know; I re-worked it a bit. I didn't want to modify the original Poller yet, so I made a copy of it. If my version of Poller is accepted, I will eventually change the Mayastor to use it.